File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/models/MusicoTest.java
package simulacionBandaMusica.models;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.factories.CantanteFactory;

import static org.junit.jupiter.api.Assertions.assertEquals;

class MusicoTest {

    @Test
    void interpretarTest() {
        Cantante cantante = CantanteFactory.create();
        String correctString = "Estoy interpretando con mi " + cantante.instrumento;
        assertEquals(correctString, cantante.interpretar());

    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/models/MusicoTest.kt
package simulacionBandaMusica.models

import org.junit.jupiter.api.Test
import simulacionBandaMusica.factories.CantanteFactory
import kotlin.test.assertEquals

internal class MusicoTest {
    @Test
    fun interpretarTest() {
        val cantante: Musico.Cantante = CantanteFactory.create()
        val correctString = "Estoy interpretando con mi " + cantante.instrumento
        assertEquals(correctString, cantante.interpretar())
    }
}